index.xml

```diff
@@ -175,6 +175,23 @@ of keys and assignment of `keyId` is out of scope for this document.
      </t>
     </section>
 
+    <section anchor="date" title="date">
```

Comment:
Hi M, thanks for your feedback!

1- on milliseconds, it could be fine, I just took the fields from signed-exchanges and I think we should somewhat converge. Sub-millisecond usually fight with latencies and time-skew (see below). 

2- iiuc created/expires are the ws-security names. To me is fine.

Reply:
Using `date` could be useful in working around this: https://github.com/w3c-dvcg/http-signatures/issues/33

Reply:
> Sub-millisecond usually fight with latencies and time-skew

Yes, but they make great monotonically increasing nonces for things like financial trading systems.

Reply:
I'd go with ``issued``, as it aligns with token-type systems like jwt (iat). This is after all the date the signature validity commenced, and is independent of the "date" of the payload. If it's important to the payload it should be encoded there. It was always problematic that we're using the default Date header, as it is even banned from requests under certain schemes and is meant to indicate the date of transmission, again separate from the document or signature date values.